src/cellEditors/FilterBox.js

```diff
@@ -82,7 +82,7 @@ var FilterBox = ComboBox.extend('FilterBox', {
             appendOptions: function(dropdown) {
                 if (!dropdown.length) {
                     // Various  operator options and/or optgroups vary per column based on `opMenu`.
-                    var opMenuGroups = this.grid.behavior.getGlobalFilter().opMenuGroups;
+                    var opMenuGroups = this.grid.behavior.filter.opMenuGroups;
```

Comment:
opMenuGroups and parserCQL seem "domain" specific. You should note that this implementation of cell editor depends on extending filterTree if you wish to use this module. Otherwise the application develop should roll their own cell editor. Subsequently, we should also consider how this will affect properties like showFilterRow. It makes an assumption about where the row should be loaded (maybe too much special meaning).

The workflow I imagine is: I'm an application developer and I want to do filtering with a provided row for filtering.
It should be load Hyper-filter or my own filter plugin
Load this cell Editor or my own 
Override getCell to specify where to render the filterCell 
If they don't want a filterrow (and opting for a different UI) they won't get one


Lastly, we can remove logic from the grid that have to make special accomodations for a filter row
@rdepena Thoughts?